fontforge/sfd.c

```diff
@@ -2782,7 +2789,7 @@ static int SFD_Dump( FILE *sfd, SplineFont *sf, EncMap *map, EncMap *normal,
 	if ( sf->grid.order2 )
 	    fprintf(sfd, "GridOrder2: %d\n", sf->grid.order2 );
 	fprintf(sfd, "Grid\n" );
-	SFDDumpSplineSet(sfd,sf->grid.splines);
```

Comment:
Wait. One other question. Why is this always false?

Reply:
The guide layer is always cubic, and usually only contains straight lines.

Reply:
Although now that you mention it I do see `sf->grid.order2`, but there's no way to change the guide layer's type in the UI so it must always be cubic.

Reply:
Changed in https://github.com/fontforge/fontforge/pull/3679/commits/4f5de27ed2a8f0ea54919aa82f0f1b39470b4329